include/micm/jit/solver/jit_solver_builder.hpp

```diff
@@ -17,7 +17,7 @@ namespace micm
   ///
   /// JIT-compiled solvers only work with vector-ordered matrices
   template<class SolverParametersPolicy, std::size_t L = MICM_DEFAULT_VECTOR_SIZE>
-  using JitSolverBuilder = SolverBuilder<
+  using JitSolverBuilder_Doolittle = SolverBuilder<
```

Comment:
```suggestion
  using DefaultJITSolverBuilder = SolverBuilder<
```

I prefer this instead of adding doolittle to the name. What do you think?

Reply:
Thanks Kyle. I do not like my current name change to the solver builder either. If I add the `Default` prefix to the solver builder as you suggested, what would be the alternative name when we add the new LU decomposition implementation?

Reply:
If it's faster, maybe `_Fast`? That'll at least indicate that something fancy is going on

Reply:
That sounds great. For consistency purpose, is it better to put both `Default` and `Fast` as prefix or suffix?

Reply:
Personally, I would just leave this as `JitSolverBuilder`. If the MOZART algorithm is faster, but someone wants to use the Doolittle algorithm (for some reason), they can just specify all the template parameters for the solver builder themselves

Reply:
Thanks @mattldawson . I just revert to the original builder name and we can do what you suggest after we implement the MOZART algorithm.